src/support/image.ts

```diff
@@ -46,9 +52,24 @@ function blurImageSVG(image: BlurImageMetadata): string {
     return `%3Csvg xmlns='http://www.w3.org/2000/svg' ${viewBox}%3E%3Cfilter id='b' color-interpolation-filters='sRGB'%3E%3CfeGaussianBlur stdDeviation='${std}'/%3E%3CfeColorMatrix values='1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 100 -1' result='s'/%3E%3CfeFlood x='0' y='0' width='100%25' height='100%25'/%3E%3CfeComposite operator='out' in='s'/%3E%3CfeComposite in2='SourceGraphic'/%3E%3CfeGaussianBlur stdDeviation='${std}'/%3E%3C/filter%3E%3Cimage width='100%25' height='100%25' x='0' y='0' preserveAspectRatio='xMidYMid slice' style='filter: url(%23b);' href='${blurDataURL}'/%3E%3C/svg%3E`
 }
 
+export function isOnlineImage(filepath: string): boolean {
+    return filepath.startsWith('http://') || filepath.startsWith('https://')
+}
+
 async function blurImageMetadata(filepath: string): Promise<BlurImageMetadata> {
     const { default: sharp } = await import('sharp')
-    const buffer = await fs.readFile(filepath)
+
+    let buffer: Buffer
+    if (isOnlineImage(filepath)) {
+        const response = await fetch(filepath)
```

Comment:
this will case BUILD too slow since each image will be fetch first and then convert to blur